### PR TITLE
Test for IGP integration

### PR DESCRIPTION
**Pull request by teolemon**
Test for IGP integration

### CODE REVIEW

File: taxonomies/categories.txt [hunk 1]
```diff
@@ -658,6 +658,30 @@ hu:Almaborok
 nl:Ambachtelijke ciders
 ro:Cidru
 
+<en:Ciders
+en:Traditional Welsh Cider
+protected_name_file_number_en: PGI-GB-01251
```
Comment (stephanegigandet) [inline taxonomies/categories.txt]: search and replace all protected_name_file_number_en: by protected_name_file_number:en:

File: taxonomies/categories.txt [hunk 2]
```diff
@@ -1117,6 +1210,36 @@ ro:Băuturi spirtoase
...
+<en:Alcoholic beverages
+en:Spirit drinks
+
+<en:Spirit drinks
+en:Spanish spirit drinks
+
+<en:Spanish spirit drinks
+es:Ronmiel de Canarias
+protected_name_file_number:en: PGI-ES-01943
+protected_name_type:en: gi
```
Comment (stephanegigandet) [inline taxonomies/categories.txt]: protected_name_type:en: gi --> should be pgi
(also in may other entries)

Comment (teolemon) [reply taxonomies/categories.txt]: it was actually GI

File: taxonomies/categories.txt (hunk 1 above)
Comment (teolemon) [reply taxonomies/categories.txt]: fixé

File: taxonomies/categories.txt [hunk 3]
```diff
@@ -36644,10 +40755,17 @@ zh:家禽
...
 country:en:France
+protected_name_file_number_en: PGI-FR-0167
+#protected_name_file_number_en: PGI-FR-0167-AM02
+protected_name_type:en: pgi
+origins:en: en:France
 
 <en:Poultries
 en:Poultries from Bresse
-fr:Volaille de Bresse
+fr:Volaille de Bresse, Poulet de Bresse, Poularde de Bresse, Chapon de Bresse
```
Comment (stephanegigandet) [inline taxonomies/categories.txt]: Poulet de Bresse, Poularde de Bresse, Chapon de Bresse  -> thouse should be children
no need to repeat the properties, they are inherited

File: taxonomies/categories.txt [hunk 4]
```diff
@@ -4465,6 +4815,77 @@ de:Bergwasser, Bergwässer
 fr:Eaux de montagne, eau de montagne
 nl:Bergwaters, Bergwater
 
+fr:marc
```
Comment (teolemon) [inline taxonomies/categories.txt]: @stephanegigandet not sure how to integrate

File: taxonomies/categories.txt [hunk 5]
```diff
@@ -11821,191 +12408,852 @@ wikidata:en:Q7566214
...
+origins:en: en:United Kingdom
+
+<en:British wines
+en:English
+protected_name_file_number:en: PDO-GB-A1585
+protected_name_type:en: pdo
+origins:en: en:United Kingdom
+
+<en:British wines
+en:Welsh
```
Comment (teolemon) [inline taxonomies/categories.txt]: will probably cause issues down the line

File: taxonomies/categories.txt [hunk 6]
```diff
@@ -11821,191 +12408,852 @@ wikidata:en:Q7566214
...
+origins:en: en:United Kingdom
+
+<en:British wines
+en:Darnibole
+protected_name_file_number:en: PDO-GB-N1636
+protected_name_type:en: pdo
+origins:en: en:United Kingdom
+
+<en:British wines
+en:English
```
Comment (teolemon) [inline taxonomies/categories.txt]: will probably cause issues down the line

File: taxonomies/categories.txt (hunk 3 above)
Comment (teolemon) [reply taxonomies/categories.txt]: fixed. I kept Volailles de Bresse as parent

File: taxonomies/categories.txt [hunk 7]
```diff
@@ -46213,6 +50907,23 @@ ciqual_food_code:en:20180
 ciqual_food_name:en:Onion, dried
 ciqual_food_name:fr:Oignon, séché
 
+<en:Vegetables
+en:French vegetables
+
+<en:French vegetables
+<en:Onions and their products
```
Comment (moon-rabbitOFF) [inline taxonomies/categories.txt]:
```suggestion
<en:Onions
```

File: taxonomies/categories.txt [hunk 8]
```diff
@@ -47620,6 +52358,13 @@ nl_be:Aardappelen
 se:Potatisar, potatis
 pnns_group_2:en:Potatoes
 
+<en:British vegetables
```
Comment (moon-rabbitOFF) [inline taxonomies/categories.txt]:
```suggestion
```

File: taxonomies/categories.txt [hunk 9]
```diff
@@ -47631,7 +52376,11 @@ fr:Pommes de terre Béa
 wikidata:en:Q2929373
 
 <fr:Pommes de terre Béa
+<en:French vegetables
```
Comment (moon-rabbitOFF) [inline taxonomies/categories.txt]:
```suggestion
```

Comment (moon-rabbitOFF) [inline taxonomies/categories.txt]:
```suggestion
```

File: taxonomies/categories.txt [hunk 10]
```diff
@@ -43340,6 +47796,206 @@ th:อาหารทะเล
...
+protected_name_type:en: pgi
+origins:en: en:United Kingdom
+
+<en:British sea products
+en:Cornish Sardines
+protected_name_file_number_en: PGI-GB-0589
+protected_name_type:en: pgi
+origins:en: en:United Kingdom
+
+<en:British sea products
```
Comment (moon-rabbitOFF) [inline taxonomies/categories.txt]:
```suggestion
<en:British sea products
<en:Smoked fishes
```

File: taxonomies/categories.txt [hunk 11]
```diff
@@ -43340,6 +47796,206 @@ th:อาหารทะเล
...
+protected_name_type:en: pdo
+origins:en: en:United Kingdom
+
+<en:British sea products
+en:Lough Neagh Eel
+protected_name_file_number_en: PGI-GB-0796
+protected_name_type:en: pgi
+origins:en: en:United Kingdom
+
+<en:British sea products
```
Comment (moon-rabbitOFF) [inline taxonomies/categories.txt]:
```suggestion
<en:British sea products
<en:Sardines
```

File: taxonomies/categories.txt [hunk 12]
```diff
@@ -43340,6 +47796,206 @@ th:อาหารทะเล
...
+protected_name_type:en: pdo
+origins:en: en:United Kingdom
+
+<en:British sea products
+en:West Wales Coracle Caught Sewin
+protected_name_file_number_en: PGI-GB-01180
+protected_name_type:en: pgi
+origins:en: en:United Kingdom
+
+<en:British sea products
```
[the remark this answers is not in the document]
Comment (teolemon) [reply taxonomies/categories.txt:47842]: fixed

File: taxonomies/categories.txt [hunk 13]
```diff
@@ -43340,6 +47796,206 @@ th:อาหารทะเล
...
+en:West Wales Coracle Caught Salmon
+protected_name_file_number_en: PGI-GB-01179
+protected_name_type:en: pgi
+origins:en: en:United Kingdom
+
+<en:British sea products
```
[the remark this answers is not in the document]
Comment (teolemon) [reply taxonomies/categories.txt:47848]: fixed

File: taxonomies/categories.txt [hunk 14]
```diff
@@ -43340,6 +47796,206 @@ th:อาหารทะเล
...
+en:Fal Oyster
+protected_name_file_number_en: PDO-GB-0885
+protected_name_type:en: pdo
+origins:en: en:United Kingdom
+
+<en:British sea products
```
[the remark this answers is not in the document]
Comment (teolemon) [reply taxonomies/categories.txt:47854]: fixed in future commit

File: taxonomies/categories.txt [hunk 15]
```diff
@@ -43340,6 +47796,206 @@ th:อาหารทะเล
...
+en:Isle of Man Queenies
+protected_name_file_number_en: PDO-GB-0856
+protected_name_type:en: pdo
+origins:en: en:United Kingdom
+
+<en:British sea products
```
[the remark this answers is not in the document]
Comment (teolemon) [reply taxonomies/categories.txt:47860]: fixed in future commit

File: taxonomies/categories.txt (hunk 11 above)
Comment (teolemon) [reply taxonomies/categories.txt:47866]: fixed in commit

File: taxonomies/categories.txt (hunk 8 above)
Comment (teolemon) [reply taxonomies/categories.txt]: I've removed the British veg alltogether.

File: taxonomies/categories.txt [hunk 16]
```diff
@@ -47644,12 +52393,22 @@ nl:Charlotte aardappel
 wikidata:en:Q1067085
 
 <en:Potatoes
+<en:French vegetables
```
[the remark this answers is not in the document]
Comment (teolemon) [reply taxonomies/categories.txt]: fixed in future commit

File: taxonomies/categories.txt (hunk 9 above)
Comment (teolemon) [reply taxonomies/categories.txt]: fixed in future commit

Comment (teolemon) [reply taxonomies/categories.txt]: fixed in future commit

File: taxonomies/categories.txt (hunk 4 above)
Comment (teolemon) [reply taxonomies/categories.txt]: fixed by adding eau de vie

File: taxonomies/categories.txt [hunk 17]
```diff
@@ -43340,6 +47796,206 @@ th:อาหารทะเล
...
+<en:Belgian sea products
+fr:Escavèche de Chimay
+protected_name_file_number_en: PGI-BE-02359
+protected_name_type:en: pgi
+origins:en: en:Belgium
+
+<en:Seafood
+en:Brazilian sea products
+
+<en:Brazilian sea products
```
[the remark this answers is not in the document]
Comment (teolemon) [reply taxonomies/categories.txt]: fixed

File: taxonomies/categories.txt (hunk 7 above)
Comment (teolemon) [reply taxonomies/categories.txt]: just fixed.

**Pull request merged**